c2cgeoportal/scaffolds/create/print/print-apps/+package+/config.yaml.mako

```diff
@@ -35,21 +35,47 @@ templates:
         processors:
         - !reportBuilder # compile all reports in current directory
             directory: '.'
-        - !configureHttpRequests
+        - !configureHttpRequests &configureHttpRequests
             httpProcessors:
             - !mapUri
+                matchers:
+                - !dnsMatch
+                    host: ${host}
                 mapping:
                     (https?)://${host}/(.*): "$1://127.0.0.1/$2"
-            - !useHttpForHttps
```

Comment:
Warning, this guy is not only changing the port, but is changing the protocol as well. Your change made the local URI redirected to https://{host}:80/... apache is going to have only one response to that: meh?